modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -9,11 +9,11 @@ endif::[]
 :_content-type: PROCEDURE
 [id="installation-extend-edge-nodes-aws-local-zones_{context}"]
 = Creating user workloads in AWS Local Zones
-After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets. 
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets.
 
-After the `openshift-installer` creates the cluster, the installation program automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+After the installation program creates the cluster, the program automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
```

Comment:
@dfitzmau what is the current/convention to refer to the installation? is`the installer` or `the installation program`? I can see both in the docs.

Reply:
The `installation program` program is correct. Some legacy instances of `openshift-installer` exist in the wider installation doc set. I'll ensure the AWS LZ docs do not contain the latter term.

Reply:
perfect, thanks!